### DISCUSSION TITLE
我做了一个有点艰难的决定 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#210] [Announcements]
https://mazhuang.org/2017/04/09/a-difficult-decision/

**Comment by codesdt** [comment]
这么优秀的讲解，我要支持你，身在北京的我已经5年多了，最近各种压力使得我还要在北京待一段时间。

**Comment by mzlogin** [comment]
@niehuaming 
加油吧少年，在哪里都要使劲把生活过成想要的样子。

**Comment by jianyifu** [comment]
很欣赏这句“庆幸的是选择的成分相对还是更多一点”，那是一种选择自己主导生活方向的积极的态度，看到壮哥写的这些话，深深地感觉到，很幸运能够认识壮哥，认识那个在生活中不断思考，沉稳靠谱的壮哥。

**Comment by mzlogin** [comment]
@jianyifu 
😄 认识你和弱鸡宝宝们我也觉得很幸运。每当看到和想到你们，我就觉得生活充满希望和精彩。有幸与你们同行一段，是在帝都最珍贵的回忆。

**Comment by xiongchengqing** [comment]
**加油！**

**Comment by mzlogin** [comment]
@xiongchengqing 

感谢，加油！

**Comment by 8077** [comment]
加油！

**Comment by mzlogin** [comment]
@8077 

谢谢鼓励，都加油生活！ 😄

**Comment by conjs** [comment]
哥们，我也也一样。离开了上海回武汉了。有时间可以喝一杯

**Comment by mzlogin** [comment]
@conjs 好啊，有时间约上 😄

**Comment by conjs** [comment]
@mzlogin 网站可以做个链接 game-coder.com

**Comment by mzlogin** [comment]
@conjs 链接已加。页面做得很别致~ 👍

**Comment by aftersss** [comment]
加油吧兄弟

**Comment by mzlogin** [comment]
@aftersss 感谢~都加油生活！

**Comment by shixinzhang** [comment]
最近也在思考能在上海待多久，纠结。。。

**Comment by mzlogin** [comment]
> @shixinzhang
> 最近也在思考能在上海待多久，纠结。。。

在还没有做出决定之前，就在上海好好生活吧~以后的事谁知道呢。保持思考，保持培养对生活的意识和品质的追求，逐渐看清自己想要什么，再做出决定。

**Comment by ehlxr** [comment]
我也是在北京打拼了好多年，最近思量着要回故乡的码农一枚，今天偶尔看到博主的文章，对 `常有一种莫名的 “临时感”` 深有体会

**Comment by mzlogin** [comment]
> @ehlxr
> 我也是在北京打拼了好多年，最近思量着要回故乡的码农一枚，今天偶尔看到博主的文章，对 `常有一种莫名的 “临时感”` 深有体会

看来我们这一代人或多或少都面临这样的处境和选择啊
